src/dialog.jsx

```diff
@@ -143,9 +143,6 @@ function getStyles(props, state) {
       width: '100%',
       textAlign: 'right',
     },
-    paper: {
```

Comment:
Isn't this property here to allow people to customize the paper background color independently?
I think that we should keep it.